synapse/models/person.py

```diff
@@ -45,9 +45,16 @@ def getModelDefs(self):
                 )),
                 ('ps:person', {}, (
                     ('dob', ('time', {}), {
-                        'doc': 'The Date of Birth (DOB) if known.',
+                        'doc': 'The date on which the person was born.',
+                    }),
+                    ('dod', ('time', {}), {
+                        'doc': 'The date on which the person died.',
                     }),
                     ('img', ('file:bytes', {}), {
+                        'deprecated': True,
```

Comment:
ps:persona:img is not marked as deprecated

Reply:
Also left off due to likely deprecating the whole form